core/src/main/java/com/example/core/components/VerticalList.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.core.components
+
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.runtime.Composable
+
+@Composable
+fun VerticalList(
```

Comment:
Why do we have to separate like this?

Reply:
I see in the design we will have screens which have the component like this